src/components/Story/Header.jsx

```diff
@@ -2,46 +2,57 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Head from 'next/head';
 import { Row, Col, Container } from 'reactstrap';
-import Imgix from '../Imgix';
 import HeaderStyle from './HeaderStyle';
+import Imgix from '../Imgix';
 
 const Header = (props) => {
   const {
     title,
+    teaser,
     primaryImgSrc,
     primaryImgCaption,
-    teaser,
   } = props;
 
+  const contents = (
+    <Container className="align-self-center">
+      <Row>
+        <Col className="post-heading">
+          <h1 className="p-2 text-white">{title}</h1>
+        </Col>
+      </Row>
+    </Container>
+  );
+
   return (
     <div className="story-header">
       <HeaderStyle />
       <Head>
         <title>{title}</title>
         <meta name="description" story={teaser} />
       </Head>
-      <Imgix tag="div" className="d-flex bg-imgIx img-fluid" src={primaryImgSrc} alt={primaryImgCaption} title={title} w="1200" h="600" fit="crop" crop="faces,edges">
-        <Container className="align-self-center">
-          <Row>
-            <Col className="post-heading">
-              <h1 className="p-2 text-white">{title}</h1>
-            </Col>
-          </Row>
-        </Container>
-      </Imgix>
+      {primaryImgSrc ? (
+        <Imgix tag="div" className="d-flex bg-imgIx img-fluid" src={primaryImgSrc} alt={primaryImgCaption} title={title} w="1200" h="600" fit="crop" crop="facial,edge">
```

Comment:
I thought the comma separated option wasn't supported? Also, weren't we going to use the focalpoint?